packages/flame/lib/src/page_render_effect.dart

```diff
@@ -0,0 +1,35 @@
+import 'dart:ui';
+
+abstract class PageRenderEffect {
+  Canvas preprocessCanvas(Canvas canvas);
+  void postprocessCanvas(Canvas canvas);
+}
+
+class PaintRenderEffect extends PageRenderEffect {
```

Comment:
Maybe this should be called something else than `Effect`, to not confuse it with an effect from the effects system.

Reply:
Agree, though I'm a bit out of ideas here...

Reply:
`Filter` maybe?

Reply:
Renamed into `Decorator`.